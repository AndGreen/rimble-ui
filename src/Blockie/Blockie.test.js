import BlockiesIdenticon from './'

describe('Blockie component sanity', () => {
  it('has name', () => {
    expect(BlockiesIdenticon.displayName).toBe('BlockiesIdenticon')
  })

  it('matches default snapshot', () => {
    const component = render(<BlockiesIdenticon />)
    expect(component).toMatchSnapshot()
  })
})
