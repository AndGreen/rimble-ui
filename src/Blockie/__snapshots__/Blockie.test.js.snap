// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blockie component sanity matches default snapshot 1`] = `
<canvas>
  
</canvas>
`;
